platform/server/src/resolvers/models.ts

```diff
@@ -7,6 +7,30 @@ export const models = [
       publishDelete: true,
     },
   },
+  {
+    name: "Product",
```

Comment:
Strange that we have so many objects just for products

Reply:
What do you mean?

Reply:
Is there ability to reuse single product type?

Reply:
I suppose you are talking about `PrefferedProduct` and `VolunteerActionProduct` ? 

If so, these are not actual product models but "join tables" between:
-  a product and a recipient  => `PrefferedProduct` maybe should be renamed to `RecipientPrefferedProduct` or something. 
- a product and a volunteer action => `VolunteerActionProduct`.


Ideally, I would have loved to just have a `@ManyToMany` annotation with graphback taking care of the join table and app developer having to declare none at.

Reply:
The annotation could be `un-direction` what I really need now or `bi-direction` - ability to define it at both ends of relationship. 

E.g in VolunteerAction model, it could be.
```

"""
Represents action that is assigned to volunteer
@model
"""
type VolunteerAction {
  ....
  """
  @ManyToMany
  """
  products: [Products]
}
```

Reply:
Let's chat about it tommorow. I think we should really disable all crud methods on them. We have 78 files changed in this PR where half of it is not going to be used at all. This is huge shortcoming of graphback that we hoping to address with @eunovo

See: https://github.com/aerogear/graphback/issues/1228